modules/webhook/WebhookModel.ts

```diff
@@ -132,6 +132,11 @@ export const WebhookModel = types
         const data = await response.json()
 
         console.log("Target executed", data)
+
+        message.set(
+          "url",
+          `https://discord.com/channels/${self.guildId}/${self.channelId}/${data.id}`,
+        )
```

Comment:
I moved the 'save' function's logic into the editor manager. But I should mention that you shouldn't set message reference after sending, as it annoyed people.